solutions/swb-reference/src/SWBStack.ts

```diff
@@ -844,7 +844,7 @@ export class SWBStack extends Stack {
   ): WorkbenchCognito {
     const props: WorkbenchCognitoProps = {
       domainPrefix: domainPrefix,
-      websiteUrl: websiteUrl,
+      websiteUrls: [websiteUrl],
```

Comment:
A couple of places needs to be updated to take in `websiteUrls` as well.
* https://github.com/awslabs/solution-spark-on-aws/blob/b045c4c0d21724d13ae7e6e92c6dda000e6e7dc6/solutions/swb-reference/src/config/example.yaml#L7
* https://github.com/awslabs/solution-spark-on-aws/blob/90a4c93ae98c85e5dd882d60116c9b5529f24c67/solutions/swb-reference/src/constants.ts#L60
   * Since the param is now an array, we can append it like what we did in line 56.

Reply:
updated